regression_em/model.py

```diff
@@ -1,18 +1,287 @@
+"""Package to perform regression EM algorithm self.
+For the detail of this algortithm,
+["Position Bias Estimation for Unbiased Learning to Rank in Personal Search" Xuanhui Wang et al.]
```

Comment:
See markdown link format
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links

Reply:
Follwed markdown link format at L3 and L25.